applications/DamApplication/python_scripts/impose_nodal_reference_temperature_process.py

```diff
@@ -15,7 +15,15 @@ def __init__(self, Model, settings ):
         model_part = Model[settings["model_part_name"].GetString()]
         variable_name = settings["variable_name"].GetString()
         initial_value = settings["initial_value"].GetDouble()
-        input_file_name = settings["input_file_name"].GetString()
+
+        # This exception is introduced to check if the parameter "input_file_name" exixts. If not, it create it and define it as "".
+        # This could be changed when the Nodal Reference Temp (input file) process will be fixed. So far, in the ACOMBO Release 6, this parameter (input_file_name) was removed from the interface.
+        try:
```

Comment:
I wouldn't use the `try` statement... You can use the `Has` method from `Parameters` object. Something like:
```py
if settings.Has("input_file_name"):
    input_file_name = settings["input_file_name"].GetString()
else:
    input_file_name = ""
```
Remember that `try` instruction tries to handle an error. Thus let's avoid it instead of tackling it ;).

Reply:
Without a doubt, your proposal seems much better than mine. I did not know the Has method from Parameters object. I still have a lot to learn ;)
Thank you very much Ruben!